python-sdk/example_dags/example_load_file.py

```diff
@@ -253,4 +253,17 @@
     )
     # [END load_file_example_19]
 
+    # [START load_file_example_20]
+    aql.load_file(
+        input_file=File(path="sftp://foo@localhost/upload/", conn_id="sftp_conn", filetype=FileType.CSV),
```

Comment:
username and hostname could be env vars, here?

Reply:
modiifed